tox.ini

```diff
@@ -1,4 +1,6 @@
 [base]
+# ignore version.py (generated by setuptool_scm) since it fails
```

Comment:
` since it fails` can you expand on this?

Reply:
This should have been removed as the `version.py` is no longer produced, I'll remove it.